src/Interceptor/Interceptor/CacheInterceptor.php

```diff
@@ -98,44 +91,47 @@ public function getSuffixPriority(): int
         return 20;
     }
 
-    private function getNamespace(Instance $instance, Cache $annotation): ?string
+    private function getNamespace(Instance $instance, Cache $annotation): string
     {
         $parameters = $instance->getMethod()
             ->getParameters()
         ;
+
         if ($annotation->getNamespace() !== null) {
-            $resolvedExpression = $this->resolveExpression(
+            return $this->resolveExpression(
```

Comment:
we are missing the `md5` here

Reply:
or we can accept it (reloading all website cache at once), but removing md5 from everywhere